test/Microsoft.ML.Tests/Scenarios/PipelineApi/TrainSaveModelAndPredict.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Models;
+using Microsoft.ML.Trainers;
+using Microsoft.ML.Transforms;
+using Xunit;
+
+namespace Microsoft.ML.Tests.Scenarios.PipelineApi
+{
+    public partial class PipelineApiScenarioTests
+    {
+        /// <summary>
+        /// Train, save/load model, predict: 
+        /// Serve the scenario where training and prediction happen in different processes (or even different machines). 
+        /// The actual test will not run in different processes, but will simulate the idea that the 
+        /// "communication pipe" is just a serialized model of some form.
+        /// </summary>
+        [Fact]
+        public async void TrainSaveModelAndPredict()
+        {
+            var dataPath = GetDataPath(SentimentDataPath);
+            var testDataPath = GetDataPath(SentimentDataPath);
+            var pipeline = new LearningPipeline();
+
+            pipeline.Add(MakeSentimentTextLoader(dataPath));
+
+            pipeline.Add(MakeSentimentTextTransform());
+
+            pipeline.Add(new FastTreeBinaryClassifier() { NumLeaves = 5, NumTrees = 5, MinDocumentsInLeafs = 2 });
+
+            pipeline.Add(new PredictedLabelColumnOriginalValueConverter() { PredictedLabelColumn = "PredictedLabel" });
+            var model = pipeline.Train<SentimentData, SentimentPrediction>();
+
+            var modelName = "trainSaveAndPredictdModel.zip";
+            DeleteOutputPath(modelName);
+            await model.WriteAsync(modelName);
+
+            var loadedModel = await PredictionModel.ReadAsync<SentimentData, SentimentPrediction>(modelName);
+            var testData = MakeSentimentTextLoader(testDataPath);
+            var evaluator = new BinaryClassificationEvaluator();
+            var metrics = evaluator.Evaluate(loadedModel, testData);
```

Comment:
remove from this test

Reply:
Move to evaluation

---
In reply to: [210042424](https://github.com/dotnet/machinelearning/pull/677#discussion_r210042424) [](ancestors = 210042424)